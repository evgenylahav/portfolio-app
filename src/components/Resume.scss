.resume-right-col {
    background: #0d0b11;
    color: white;
    padding: 2em;
}
.resume-right-col hr {
    border-top: 3px solid rgb(114, 1, 1);
    width: 90%;
    margin: auto;
}
.resume-left-col hr {
    border-top: 3px solid rgb(9, 116, 57);
    width: 50%;
    margin: auto;
}

