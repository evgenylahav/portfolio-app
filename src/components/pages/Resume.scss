.sidebar-section img {
    width: 100%;
    margin-bottom: 2em;
}

.resume-header-image {
    width: 100%;
    max-height: 500px;
    margin-top: -2em;
    margin-bottom: 2em;
}

.resume-right-col {
    background: #0d0b11;
    color: white;
    padding: 2em;
}
.resume-right-col hr {
    border-top: 3px solid rgb(114, 1, 1);
    width: 90%;
    margin: auto;
}
.resume-left-col hr {
    border-top: 3px solid rgb(9, 116, 57);
    width: 50%;
    margin: auto;
}

.resume-container h3{
    font-family: 'Indie Flower', cursive;
    font-size: 30px;
    color: black;
    font-weight: bold;
    text-align: center;
    width: 75%;
}

.verticalLine {
    border-left: solid rgb(134, 139, 137);;
}

.resume-profile-pic {
    float: left;
    margin-top: 30px;
    width: 30%;
}