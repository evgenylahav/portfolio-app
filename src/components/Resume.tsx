import * as React from 'react';
import { Container, Row, Col, Image } from 'react-bootstrap';
import { Grid, Cell } from 'react-mdl';
import { ResumeLine } from './ResumeLine';
import { Skills } from './Skills';
import board from '../assets/blackboard-chalk.jpg';
import { Social } from './Social';
import im1 from "../assets/Image-1.png";
import './Resume.scss';

export interface ResumeCompProps {}
export interface ResumeCompState {}

export class Resume extends React.Component<ResumeCompProps, ResumeCompState> {
    constructor(props: any) {
        super(props);
        this.state = {};
    }

    public render() {
        return (
            <div>
                {/* <Image src={board} className="resume-header-image" /> */}
                <Container className='resume-container'>
                    <h3>My Resume</h3>
                    <Row>
                        <Col xs={12} sm={4}>
                            <div style={{textAlign: 'center'}}>
                                <Image src={im1} roundedCircle className="resume-profile-pic" />

                            </div>
                            <h2 style={{textAlign: 'center'}}>Evgeny Lahav</h2>
                            <h4 style={{color: 'grey', textAlign: 'center'}}>Software Engineer</h4>                 
                            <hr/>
                            <p style={{paddingTop: '5px'}}>An experienced and enthuziastic SW engineer, specializing in Python, JavaScript and React</p>
                            <hr/>
                            <h5>email</h5>
                            <p>evgeny.lahav@gmail.com</p>
                            <h5>mobile</h5>
                            <p>+972-(0)5444-86872</p>
                            <h5>website</h5>
                            <p>...</p>
                            <hr/>
                        </Col>
                        <Col sm={1} className='verticalLine' />
                        
                        <Col xs={12} sm={7}>
                            <h2>Experience</h2>
                            <ResumeLine 
                                startYear='2013' 
                                endYear='current' 
                                placeName='Applied Materials'
                                description='Software engineer: ...'
                            />
                            <ResumeLine 
                                startYear='2007' 
                                endYear='2013' 
                                placeName='Applied Materials'
                                description='System engineer: ...'
                            />
                            <ResumeLine 
                                startYear='2006' 
                                endYear='2007' 
                                placeName='Applied Materials'
                                description='Software QA engineer: ...'
                            />
                            <hr/>
                            <h2>Education</h2>
                            <ResumeLine 
                                startYear='2014' 
                                endYear='2018' 
                                placeName='Ben-Gurion University'
                                description='M.Sc in Electro-optical engineering. Grades average of 90. Thesis subject...'
                            />
                            <ResumeLine 
                                startYear='2001' 
                                endYear='2005' 
                                placeName='Ben-Gurion University'
                                description='B.Sc in Bio-Medical Engineering. Grades average of 85.'
                            />
                            <hr/>
                            <h2>Skills</h2>
                            <Skills skill='Python' skillLevel={70}/>
                            <Skills skill='JavaScript' skillLevel={50}/>
                            <Skills skill='React' skillLevel={40}/>
                            <Skills skill='Scientific Programming' skillLevel={80}/>
                            <Skills skill='Clean Code' skillLevel={60}/>
                        </Col>

                    </Row>
                    <Social />
                </Container>
            </div>
        );
    }
}
                    /* <Cell className='resume-left-col' col={4}>
                        
                    </Cell>
                    <Cell className='resume-right-col'  col={8}>
                        
                    </Cell>
                </Grid> */
            // </div>
    //     );
    // }
// }