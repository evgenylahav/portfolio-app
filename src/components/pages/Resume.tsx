import * as React from 'react';
import { Container, Row, Col, Image } from 'react-bootstrap';
import { ResumeLine } from '../common/ResumeLine';
import { Skills } from '../common/Skills';
import im1 from "../../assets/Image-1.png";
import bgu from '../../assets/bgu.png';
import amat from '../../assets/amat.jpg'
import './Resume.scss';

export interface ResumeCompProps {}
export interface ResumeCompState {}

export class Resume extends React.Component<ResumeCompProps, ResumeCompState> {
    constructor(props: any) {
        super(props);
        this.state = {};
    }

    public render() {
        return (
            <div>
                <Container className='resume-container'>
                    <h3>My Resume</h3>
                    <Row>
                        <Col xs={12} sm={4}>
                            <div style={{textAlign: 'center'}}>
                                <Image src={im1} roundedCircle className="resume-profile-pic" />
                            </div>
                            <h3 style={{textAlign: 'center'}}>Evgeny Lahav</h3>
                            <h4 style={{color: 'grey', textAlign: 'center'}}>Software Engineer</h4>                 
                            <hr/>
                            <p style={{paddingTop: '5px'}}>An experienced and enthuziastic SW engineer, specializing in Python, JavaScript and React</p>
                            <hr/>
                            <h5>email</h5>
                            <p>evgeny.lahav@gmail.com</p>
                            <h5>mobile</h5>
                            <p>+972-(0)5444-86872</p>
                            <h5>website</h5>
                            <p>...</p>
                            <hr/>
                            <h3>Skills</h3>
                            <Skills skill='Python' skillLevel={70}/>
                            <Skills skill='JavaScript' skillLevel={55}/>
                            <Skills skill='React' skillLevel={65}/>
                            <Skills skill='Scientific Programming' skillLevel={85}/>
                            <Skills skill='SW Design and Architecture' skillLevel={60}/>
                            <Skills skill='Clean Code' skillLevel={80}/>
                            <hr/>
                            <h3>Side Projects</h3>
                            <ul>
                                <li>Personal portfolio website</li>
                                <li><a href='https://exercism.io/' target="_blank">Excersicm: Python, JavaScript</a></li>
                                <li><a href='https://www.100daysofcode.com/' target="_blank">100 Days Of Code</a></li>
                            </ul>

                        </Col>
                        <Col sm={1} className='verticalLine' />
                        
                        <Col xs={12} sm={7}>
                            <h3>Experience</h3>
                            <ResumeLine 
                                startYear='2016' 
                                endYear='current' 
                                placeName='Applied Materials'
                                img={amat}
                                position='Senior Software engineer'
                                description={[`Senior Software engineer in the system SW group. The group is responsible for providing quick 
                                               and quality solutions to support the calibration process of the tools.`,
                                              `My responsibility was leading the transformation of the group technology from development
                                               in scientific programming languages, to modern full-stack development with front-end in React js and
                                               back-end in Java and Python`,
                                              'Leading training and educational activities for the group by writing a weekly blog, preparing workshops and trainings of different sorts',
                                              `Documentation and implementation of the SW development concepts as defined by the SW architecture group:
                                               CI pipeline, Git components, Modular project structures, etc...`]}
                            />
                            <ResumeLine 
                                startYear='2013' 
                                endYear='2016' 
                                placeName='Applied Materials'
                                img={amat}
                                position='Software engineer'
                                description={['Software engineer in the apps and protptypes team',
                                              'Responsible for developing applicationsand calibrations for the Algorithms, System and Applications groups',
                                              'The development was done with scientific programming tools and concepts - mainly with Python and Matlab']}
                            />
                            <ResumeLine 
                                startYear='2007' 
                                endYear='2013' 
                                placeName='Applied Materials'
                                img={amat}
                                position='System engineer'
                                description={['Responsible for the development side of a highly sophisticated multi-disciplinary tool',
                                               'Defining and executing C&Fs and feasibility tests',
                                               'Defining SW requirements specifications',
                                               'Developing analytic tools for data analysis and tool calibration']}
                            />
                            <ResumeLine 
                                startYear='2006' 
                                endYear='2007' 
                                placeName='Applied Materials'
                                img={amat}
                                position='Software QA engineer'
                                description={['Execution of manual SW qualification', 'Leading a team of 5 testers']}
                            />
                            <hr/>
                            <h3>Education</h3>
                            <ResumeLine 
                                startYear='2014' 
                                endYear='2018' 
                                placeName='Ben-Gurion University'
                                position='M.Sc in Electro-optical engineering'
                                img={bgu}
                                description={['Grades average of 90', 'Specialization - optical computing',
                                              `Thesis subject - new optical encryption system based on encryption of information of the Fourier magnitude
                                               creating a non-linear and safe encryption system, which is simple for implementation and hard for hacking`]}
                            />
                            <ResumeLine 
                                startYear='2001' 
                                endYear='2005' 
                                placeName='Ben-Gurion University'
                                position='B.Sc in Bio-Medical Engineering'
                                img={bgu}
                                description={['Grades average of 85', 'Specialization - electronics and image processing']}
                            />
                            <hr/>
                        </Col>
                    </Row>
                </Container>
            </div>
        );
    }
}