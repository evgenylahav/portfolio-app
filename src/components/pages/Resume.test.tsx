import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { Resume } from './Resume';
import { shallow } from 'enzyme';
import { testNameToKey } from 'jest-snapshot/build/utils';


describe('AboutMe unit test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Resume />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('checks that only one about me page is created', () => {
    const element = shallow(
      <Resume />
    );
    expect(element.find(`[data-test='resume-page']`).length).toBe(1);
  });

  test('has a valid snapshot', () => {
    const component = renderer.create(
      <Resume />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
