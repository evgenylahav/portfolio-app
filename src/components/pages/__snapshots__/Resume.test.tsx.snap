// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMe unit test has a valid snapshot 1`] = `
<div
  data-test="resume-page"
>
  <div
    className="resume-container container"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-4 col-12"
      >
        <div
          style={
            Object {
              "textAlign": "center",
            }
          }
        >
          <img
            className="resume-profile-pic rounded-circle"
            src="Image-1.png"
          />
        </div>
        <h3
          style={
            Object {
              "textAlign": "center",
            }
          }
        >
          Evgeny Lahav
        </h3>
        <h4
          style={
            Object {
              "color": "grey",
              "textAlign": "center",
            }
          }
        >
          Software Engineer
        </h4>
        <hr />
        <p
          style={
            Object {
              "paddingTop": "5px",
            }
          }
        >
          An experienced and enthuziastic SW engineer, specializing in Python, JavaScript and React
        </p>
        <hr />
        <h5>
          email
        </h5>
        <p>
          evgeny.lahav@gmail.com
        </p>
        <h5>
          mobile
        </h5>
        <p>
          +972-(0)5444-86872
        </p>
        <h5>
          website
        </h5>
        <p>
          <a
            href="https://evgenylahav.github.io/portfolio-app/"
          >
            https://evgenylahav.github.io/portfolio-app/
          </a>
        </p>
        <hr />
        <h3>
          Skills
        </h3>
        <div
          className="container"
          data-test="skills-container"
        >
          <div
            className="col-sm-8 col-12"
          >
            <div
              style={
                Object {
                  "display": "flex",
                }
              }
            >
              Python
            </div>
          </div>
          <div
            className="col-sm-12 col-12"
          >
            <div>
              <div
                className="progress"
                style={
                  Object {
                    "color": "green",
                    "marginBottom": "20px",
                  }
                }
              >
                <div
                  aria-valuemax={100}
                  aria-valuemin={0}
                  aria-valuenow={70}
                  className="progress-bar"
                  role="progressbar"
                  style={
                    Object {
                      "width": "70%",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
        <div
          className="container"
          data-test="skills-container"
        >
          <div
            className="col-sm-8 col-12"
          >
            <div
              style={
                Object {
                  "display": "flex",
                }
              }
            >
              JavaScript
            </div>
          </div>
          <div
            className="col-sm-12 col-12"
          >
            <div>
              <div
                className="progress"
                style={
                  Object {
                    "color": "green",
                    "marginBottom": "20px",
                  }
                }
              >
                <div
                  aria-valuemax={100}
                  aria-valuemin={0}
                  aria-valuenow={55}
                  className="progress-bar"
                  role="progressbar"
                  style={
                    Object {
                      "width": "55%",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
        <div
          className="container"
          data-test="skills-container"
        >
          <div
            className="col-sm-8 col-12"
          >
            <div
              style={
                Object {
                  "display": "flex",
                }
              }
            >
              React
            </div>
          </div>
          <div
            className="col-sm-12 col-12"
          >
            <div>
              <div
                className="progress"
                style={
                  Object {
                    "color": "green",
                    "marginBottom": "20px",
                  }
                }
              >
                <div
                  aria-valuemax={100}
                  aria-valuemin={0}
                  aria-valuenow={65}
                  className="progress-bar"
                  role="progressbar"
                  style={
                    Object {
                      "width": "65%",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
        <div
          className="container"
          data-test="skills-container"
        >
          <div
            className="col-sm-8 col-12"
          >
            <div
              style={
                Object {
                  "display": "flex",
                }
              }
            >
              Scientific Programming
            </div>
          </div>
          <div
            className="col-sm-12 col-12"
          >
            <div>
              <div
                className="progress"
                style={
                  Object {
                    "color": "green",
                    "marginBottom": "20px",
                  }
                }
              >
                <div
                  aria-valuemax={100}
                  aria-valuemin={0}
                  aria-valuenow={85}
                  className="progress-bar"
                  role="progressbar"
                  style={
                    Object {
                      "width": "85%",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
        <div
          className="container"
          data-test="skills-container"
        >
          <div
            className="col-sm-8 col-12"
          >
            <div
              style={
                Object {
                  "display": "flex",
                }
              }
            >
              SW Design and Architecture
            </div>
          </div>
          <div
            className="col-sm-12 col-12"
          >
            <div>
              <div
                className="progress"
                style={
                  Object {
                    "color": "green",
                    "marginBottom": "20px",
                  }
                }
              >
                <div
                  aria-valuemax={100}
                  aria-valuemin={0}
                  aria-valuenow={60}
                  className="progress-bar"
                  role="progressbar"
                  style={
                    Object {
                      "width": "60%",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
        <div
          className="container"
          data-test="skills-container"
        >
          <div
            className="col-sm-8 col-12"
          >
            <div
              style={
                Object {
                  "display": "flex",
                }
              }
            >
              Clean Code
            </div>
          </div>
          <div
            className="col-sm-12 col-12"
          >
            <div>
              <div
                className="progress"
                style={
                  Object {
                    "color": "green",
                    "marginBottom": "20px",
                  }
                }
              >
                <div
                  aria-valuemax={100}
                  aria-valuemin={0}
                  aria-valuenow={80}
                  className="progress-bar"
                  role="progressbar"
                  style={
                    Object {
                      "width": "80%",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
        <hr />
        <h3>
          Side Projects
        </h3>
        <ul>
          <li>
            <a
              href="/portfolio-app"
              rel="noopener noreferrer"
              target="_blank"
            >
              Personal portfolio website
            </a>
          </li>
          <li>
            <a
              href="https://exercism.io/"
              rel="noopener noreferrer"
              target="_blank"
            >
              Excersicm: Python, JavaScript
            </a>
          </li>
          <li>
            <a
              href="https://www.100daysofcode.com/"
              rel="noopener noreferrer"
              target="_blank"
            >
              100 Days Of Code
            </a>
          </li>
        </ul>
      </div>
      <div
        className="verticalLine col-sm-1"
      />
      <div
        className="col-sm-7 col-12"
      >
        <h3>
          Experience
        </h3>
        <div
          className="container"
          data-test="resume-line-container"
        >
          <div
            className="row"
          >
            <div
              className="col-sm-3 col-12"
            >
              <p
                style={
                  Object {
                    "fontSize": "18px",
                    "paddingTop": "5px",
                  }
                }
              >
                2016
                 - 
                current
              </p>
            </div>
            <div
              className="col-sm-9 col-12"
            >
              <img
                className="resume-place-name-image img-thumbnail"
                src="amat.jpg"
              />
              <h4
                style={
                  Object {
                    "marginTop": "0px",
                  }
                }
              >
                Applied Materials
              </h4>
              <h6
                style={
                  Object {
                    "fontWeight": "bold",
                  }
                }
              >
                Senior Software engineer
              </h6>
              <ul>
                 
                <li>
                  Senior Software engineer in the system SW group. The group is responsible for providing quick 
                                               and quality solutions to support the calibration process of the tools.
                </li>
                <li>
                  My responsibility was leading the transformation of the group technology from development
                                               in scientific programming languages, to modern full-stack development with front-end in React js and
                                               back-end in Java and Python
                </li>
                <li>
                  Leading training and educational activities for the group by writing a weekly blog, preparing workshops and trainings of different sorts
                </li>
                <li>
                  Documentation and implementation of the SW development concepts as defined by the SW architecture group:
                                               CI pipeline, Git components, Modular project structures, etc...
                </li>
                 
              </ul>
            </div>
          </div>
        </div>
        <div
          className="container"
          data-test="resume-line-container"
        >
          <div
            className="row"
          >
            <div
              className="col-sm-3 col-12"
            >
              <p
                style={
                  Object {
                    "fontSize": "18px",
                    "paddingTop": "5px",
                  }
                }
              >
                2013
                 - 
                2016
              </p>
            </div>
            <div
              className="col-sm-9 col-12"
            >
              <img
                className="resume-place-name-image img-thumbnail"
                src="amat.jpg"
              />
              <h4
                style={
                  Object {
                    "marginTop": "0px",
                  }
                }
              >
                Applied Materials
              </h4>
              <h6
                style={
                  Object {
                    "fontWeight": "bold",
                  }
                }
              >
                Software engineer
              </h6>
              <ul>
                 
                <li>
                  Software engineer in the apps and protptypes team
                </li>
                <li>
                  Responsible for developing applicationsand calibrations for the Algorithms, System and Applications groups
                </li>
                <li>
                  The development was done with scientific programming tools and concepts - mainly with Python and Matlab
                </li>
                 
              </ul>
            </div>
          </div>
        </div>
        <div
          className="container"
          data-test="resume-line-container"
        >
          <div
            className="row"
          >
            <div
              className="col-sm-3 col-12"
            >
              <p
                style={
                  Object {
                    "fontSize": "18px",
                    "paddingTop": "5px",
                  }
                }
              >
                2007
                 - 
                2013
              </p>
            </div>
            <div
              className="col-sm-9 col-12"
            >
              <img
                className="resume-place-name-image img-thumbnail"
                src="amat.jpg"
              />
              <h4
                style={
                  Object {
                    "marginTop": "0px",
                  }
                }
              >
                Applied Materials
              </h4>
              <h6
                style={
                  Object {
                    "fontWeight": "bold",
                  }
                }
              >
                System engineer
              </h6>
              <ul>
                 
                <li>
                  Responsible for the development side of a highly sophisticated multi-disciplinary tool
                </li>
                <li>
                  Defining and executing C&Fs and feasibility tests
                </li>
                <li>
                  Defining SW requirements specifications
                </li>
                <li>
                  Developing analytic tools for data analysis and tool calibration
                </li>
                 
              </ul>
            </div>
          </div>
        </div>
        <div
          className="container"
          data-test="resume-line-container"
        >
          <div
            className="row"
          >
            <div
              className="col-sm-3 col-12"
            >
              <p
                style={
                  Object {
                    "fontSize": "18px",
                    "paddingTop": "5px",
                  }
                }
              >
                2006
                 - 
                2007
              </p>
            </div>
            <div
              className="col-sm-9 col-12"
            >
              <img
                className="resume-place-name-image img-thumbnail"
                src="amat.jpg"
              />
              <h4
                style={
                  Object {
                    "marginTop": "0px",
                  }
                }
              >
                Applied Materials
              </h4>
              <h6
                style={
                  Object {
                    "fontWeight": "bold",
                  }
                }
              >
                Software QA engineer
              </h6>
              <ul>
                 
                <li>
                  Execution of manual SW qualification
                </li>
                <li>
                  Leading a team of 5 testers
                </li>
                 
              </ul>
            </div>
          </div>
        </div>
        <hr />
        <h3>
          Education
        </h3>
        <div
          className="container"
          data-test="resume-line-container"
        >
          <div
            className="row"
          >
            <div
              className="col-sm-3 col-12"
            >
              <p
                style={
                  Object {
                    "fontSize": "18px",
                    "paddingTop": "5px",
                  }
                }
              >
                2014
                 - 
                2018
              </p>
            </div>
            <div
              className="col-sm-9 col-12"
            >
              <img
                className="resume-place-name-image img-thumbnail"
                src="bgu.png"
              />
              <h4
                style={
                  Object {
                    "marginTop": "0px",
                  }
                }
              >
                Ben-Gurion University
              </h4>
              <h6
                style={
                  Object {
                    "fontWeight": "bold",
                  }
                }
              >
                M.Sc in Electro-optical engineering
              </h6>
              <ul>
                 
                <li>
                  Grades average of 90
                </li>
                <li>
                  Specialization - optical computing
                </li>
                <li>
                  Thesis subject - new optical encryption system based on encryption of information of the Fourier magnitude
                                               creating a non-linear and safe encryption system, which is simple for implementation and hard for hacking
                </li>
                 
              </ul>
            </div>
          </div>
        </div>
        <div
          className="container"
          data-test="resume-line-container"
        >
          <div
            className="row"
          >
            <div
              className="col-sm-3 col-12"
            >
              <p
                style={
                  Object {
                    "fontSize": "18px",
                    "paddingTop": "5px",
                  }
                }
              >
                2001
                 - 
                2005
              </p>
            </div>
            <div
              className="col-sm-9 col-12"
            >
              <img
                className="resume-place-name-image img-thumbnail"
                src="bgu.png"
              />
              <h4
                style={
                  Object {
                    "marginTop": "0px",
                  }
                }
              >
                Ben-Gurion University
              </h4>
              <h6
                style={
                  Object {
                    "fontWeight": "bold",
                  }
                }
              >
                B.Sc in Bio-Medical Engineering
              </h6>
              <ul>
                 
                <li>
                  Grades average of 85
                </li>
                <li>
                  Specialization - electronics and image processing
                </li>
                 
              </ul>
            </div>
          </div>
        </div>
        <hr />
      </div>
    </div>
  </div>
</div>
`;
